src/openrct2/scripting/ScMap.hpp

```diff
@@ -0,0 +1,206 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2018 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#ifdef __ENABLE_SCRIPTING__
+
+#    include "../common.h"
+#    include "../ride/Ride.h"
+#    include "../world/Map.h"
+#    include "Duktape.hpp"
+#    include "ScEntity.hpp"
+#    include "ScRide.hpp"
+#    include "ScTile.hpp"
+
+namespace OpenRCT2::Scripting
+{
+    class ScMap
```

Comment:
Class is using snake_case should be CamelCase

Probably should be named `ScriptMap` (file as well)